roles/nfs/tasks/server.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: install nfs server (redhat)
+  yum: "update_cache=yes name={{ item }}"
+  with_items:
+    - nfs-utils
+    - libnfsidmap
+  when: ansible_os_family == "RedHat"
+
+- name: install nfs server (debian)
+  apt: "update_cache=yes name={{ item }}"
+  with_items:
+    - nfs-kernel-server
+    - rpcbind
+  when: ansible_os_family == "Debian"
+
+- name: enable nfs server services (redhat)
+  service: "name={{item}} enabled=yes"
```

Comment:
NFS servers are referenced directly, there is no auto-discovery, so there is no issue with master/slaves.